docs/en/sim_xplane/README.md

```diff
@@ -0,0 +1,214 @@
+# X-Plane SITL Simulation
+
+X-Plane is a comprehensive flight simulator that can be used with PX4 for Software-In-The-Loop (SITL) simulation. This integration allows testing PX4 with highly realistic flight dynamics across multiple vehicle types.
```

Comment:
```suggestion
:::warning
This simulator is [community supported and maintained](../simulation/community_supported_simulators.md).
It may or may not work with current versions of PX4 and may be removed in future releases.

See [Toolchain Installation](../dev_setup/dev_env.md) for information about the environments and tools supported by the core development team.
:::

X-Plane is a comprehensive flight simulator that can be used with PX4 for Software-In-The-Loop (SITL) simulation.
This integration allows testing PX4 with highly realistic flight dynamics across multiple vehicle types.
```